homeassistant/components/zha/core/channels/security.py

```diff
@@ -218,7 +218,7 @@ def _panic(self) -> None:
             IAS_ACE_PANIC,
         )
 
-    def _set_alarm(self, status: AceCluster.PanelStatus, event: str) -> None:
+    def _set_alarm(self, status: AceCluster.AlarmStatus, event: int) -> None:
```

Comment:
The event argument is not used at all. Should it be dropped?

Reply:
Dave is better equipped to answer this.
Could be this is coming from the cluster command and argument is present in the handler, even if discarded later.